assets/json/webnn_status.json

```diff
@@ -1374,9 +1374,9 @@
       ],
       "dml_progress": 4,
       "dml_chromium_version_added": "M122",
-      "coreml_op": [""],
-      "coreml_progress": 2,
-      "coreml_chromium_version_added": "",
+      "coreml_op": ["linear"],
```

Comment:
this is done with ` add(mul(alpha, a), beta)` instead.

Reply:
Thanks, fixed.